From: Michael Froman <mjfroman@mac.com>
Date: Wed, 29 Nov 2023 12:20:21 -0600
Subject: (tmp-cherry-pick) Revert "Remove device_status dimensions for perf
 bot" (ffd656694a)

This reverts commit cb5f65054538ee3c17f4c73799ffb89c515c9f4f.

Reason for revert: Bot check now runs that flips this dimension on the bot so we can now actually use this dimension.

Original change's description:
> Remove device_status dimensions for perf bot
>
> Bug: b/299080080
> Change-Id: I647d9fae65593d7c78e124278c085be8d9c1b61b
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/319860
> Reviewed-by: Jeremy Leconte <jleconte@google.com>
> Commit-Queue: Christoffer Jansson <jansson@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#40729}

Bug: b/299080080
Change-Id: I2ce3da17e182fb9428e022ce540aa51fe417bcd9
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/319883
Reviewed-by: Jeremy Leconte <jleconte@google.com>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Christoffer Jansson <jansson@google.com>
Cr-Commit-Position: refs/heads/main@{#40735}
---
 infra/specs/internal.client.webrtc.json | 1 +
 infra/specs/mixins.pyl                  | 3 ++-
 infra/specs/mixins_webrtc.pyl           | 1 +
 3 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/infra/specs/internal.client.webrtc.json b/infra/specs/internal.client.webrtc.json
index 7f23682b12..67033dff3f 100644
--- a/infra/specs/internal.client.webrtc.json
+++ b/infra/specs/internal.client.webrtc.json
@@ -485,6 +485,7 @@
             }
           ],
           "dimensions": {
+            "device_status": "available",
             "id": "mac-254-e504",
             "os": "iOS-15.7.8",
             "pool": "WebRTC"
diff --git a/infra/specs/mixins.pyl b/infra/specs/mixins.pyl
index 57855eae82..1fcb3d206e 100644
--- a/infra/specs/mixins.pyl
+++ b/infra/specs/mixins.pyl
@@ -79,7 +79,8 @@
       'dimensions': {
         'os': 'iOS-15.7.8',
         'pool': 'WebRTC',
-        'id': 'mac-254-e504'
+        'id': 'mac-254-e504',
+        'device_status': 'available'
       }
     }
   },
diff --git a/infra/specs/mixins_webrtc.pyl b/infra/specs/mixins_webrtc.pyl
index ea871d79f2..ded32c10e3 100644
--- a/infra/specs/mixins_webrtc.pyl
+++ b/infra/specs/mixins_webrtc.pyl
@@ -60,6 +60,7 @@
         'os': 'iOS-15.7.8',
         'pool': 'WebRTC',
         'id': 'mac-254-e504',
+        'device_status': 'available'
       },
     },
   },
